authMiddleware.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = (req, res, next) => {
+  const token = req.header('auth-token');
+  if (!token) return res.status(401).send('Access Denied');
+
+  try {
+    const verified = jwt.verify(token, process.env.TOKEN_SECRET);
+    req.user = verified;
```

Comment:
The secret used to verify the token is directly accessed from 'process.env'. It would be better to have a config file where all environment variables are accessed and exported from. This adds an extra layer of security and centralizes the access to environment variables.